include/openthread/thread_ftd.h

```diff
@@ -67,10 +67,13 @@ typedef struct
     int8_t       mLastRssi;             ///< Last observed RSSI
     uint16_t     mFrameErrorRate;       ///< Frame error rate (0xffff->100%). Requires error tracking feature.
     uint16_t     mMessageErrorRate;     ///< (IPv6) msg error rate (0xffff->100%). Requires error tracking feature.
+    uint16_t     mQueuedMessageCnt;     ///< Number of queued indirect messages for the child.
```

Comment:
1) Suggest putting "indirect" in the field name.
```suggestion
    uint16_t     mIndirectQueueCnt;     ///< Number of queued indirect messages for the child.
```
2) IEEE 802.15.4-2015 does not count queued CSL transmissions as being in the indirect queue.

Reply:
What I intended to show in this field is how many packets are queued. This could be packets not sent due to CSL sample not reached or data poll not received yet.

Reply:
Can we update the comment to match what was intended?

Reply:
Updated